.github/workflows/ci.yaml

```diff
@@ -15,8 +15,19 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: "1.17.0"
-      - name: make lint
-        run: make lint
+      - name: make verify
+        run: make verify
```

Comment:
Hey @tstromberg I modified this so it checks more than just lint steps. I think its in line with the idea behind lint-install's `lint`.